src/youtube-player/yt.d.ts

```diff
@@ -0,0 +1,973 @@
+// Type definitions for YouTube
```

Comment:
Can we reference this as a dependency? Copy-pasting code from somewhere else (even a `d.ts`) isn't allowed; it either needs to be a dep or put into a `third_party` directory with appropriate license information